javascript-loops/main.js

```diff
@@ -0,0 +1,77 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+
+// getNumbersToTen Function
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 1;
+  while (currentNumber <= 10) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  }
+  return numbers;
+}
+
+// getEvenNumbersToTwenty
+function getEvenNumbersToTwenty() {
+  var evenNumbers = [];
+  var currentNumber = 2;
+  while (currentNumber <= 20) {
+    evenNumbers.push(currentNumber);
+    currentNumber += 2;
+  }
+  return evenNumbers;
+}
+
+// getNumbersToTen function
+function repeatWord(word, times) {
+  var count = 1;
+  var repeated = '';
+  while (count < times) {
+    repeated += word;
+    count++;
+  }
+  return repeated;
+}
+
+// logEachCharacter function
+function logEachCharacter(string) {
+  var i = 0;
+  for (i = 0; i < string.length; i++) {
```

Comment:
You can either define var `i` in the expression or you can do something like this:
```
var i = 0;
for (; i < string.length; i++) {
```